src/Tes.RunnerCLI/Commands/CommandHandlers.cs

```diff
@@ -23,14 +26,13 @@ internal class CommandHandlers
             {
                 var options = BlobPipelineOptionsConverter.ToBlobPipelineOptions(blockSize, writers, readers, bufferCapacity, apiVersion);
 
-                await ExecuteTransferAsSubProcessAsync(CommandFactory.DownloadCommandName, file, options);
+                await ExecuteTransferAsSubProcessAsync(CommandFactory.DownloadCommandName, file, options, false);
```

Comment:
This code path should not be executed when using the `upload` or `download` command, so I am not sure why this change here. This is only executed when using the root `exec` command which we are not using now.